versions/3.0.md

```diff
@@ -786,9 +734,89 @@ schema:
   type: file
 ```
 
+#### <a name="requestBodyObject"></a>Request Body Object
+
+Describes a single request body.
+
+If the `requestBody` attribute is not present, or `schema` is not defined, the request body is not required.
```

Comment:
@IvanGoncharov this concept is meant to preclude the need for `required`. If no request body is needed, then one should not be specified. You could also specify a schema with nothing specified and get close the same net effect.